test/test_helper.go

```diff
@@ -117,6 +117,27 @@ func AssertS3BucketPublicAccessBlockConfigurationEnabled(t *testing.T, terraform
 	}
 }
 
+func AssertS3BucketPublicAccessBlockConfigurationDisabled(t *testing.T, terraformOptions *terraform.Options) {
+	region := terraformOptions.EnvVars["AWS_DEFAULT_REGION"]
+	s3Client, err := aws.NewS3ClientE(t, region)
+	if err != nil {
+		assert.FailNow(t, "Error creating s3client")
+		return
+	}
+
+	bucketName := terraformOptions.Vars["test_name"].(string)
+	params := &s3.GetPublicAccessBlockInput{
+		Bucket: awssdk.String(bucketName),
+	}
+
+	_, err = s3Client.GetPublicAccessBlock(params)
+
+	if err != nil {
+		return
+	}
+	assert.FailNow(t, "Public access block created and should not have been")
```

Comment:
Did you test this out and get it to fail, Cory? It sounds like from this, if it errors out we assume it works right? I guess my thought is, what if it fails for some other reason entirely? I feel like we want to make sure it errors out with the *right* error, maybe?

Or maybe I'm just being way too nitpicky for a relatively minor change. :)

Reply:
I think it's fair. How about using [assert.Error](https://pkg.go.dev/github.com/stretchr/testify/assert#Error) and capture the expected error so we don't accidentally catch and ignore an unexpected error.